.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing
+This project only accepts features and bug fixes that have been discussed in an issue. The issue must have had input by 
+at least one of the primary developers [spaulaus](https://github.com/spaulaus) or [ksmith0](https://github.com/ksmith0). 
+
+All interactions with this project must follow the 
+[Code of Conduct](https://github.com/spaulaus/paass-laughing-conqueror/blob/master/.github/CODE_OF_CONDUCT.md).
+
+# Where We Need Help
+* Unit Testing
+* Documentation
+  * Wiki (formatting / content updates)
+  * How To Articles
+  * Code Documentation
+
+# Pull Request Process
```

Comment:
Thanks for posting this. I seem to always forget what we had decided upon.

Reply:
This is more for external contributors. I need to flesh this out more generally to encompass how I typically operate.